infra/gcp/ensure-prod-storage.sh

```diff
@@ -359,6 +359,12 @@ color 6 "Handling special cases"
             $(svc_acct_email "${GCR_BACKUP_TEST_PRODBAK_PROJECT}" "${PROMOTER_SVCACCT}")
     done
 
+    # Special case: empower k8s-infra-gcs-access-logs@kubernetes.io to read k8s-artifacts-gcslogs
+    # k8s-artifacts-gcslogs receive and store Cloud Audit logs for k8s-artificats-prod.
+    ensure_gcs_role_binding "gs://k8s-artifacts-gcslogs" \
```

Comment:
Is this sufficient?  Don't we also want to import into BQ ?

Reply:
@thockin I prefer to leave them the choice of the analysis tool. I would take care of it if there is a need to use BQ.